examples/create_libraries/example/hw_example.dart

```diff
@@ -0,0 +1,5 @@
+import 'package:create_libraries/hw_mp.dart';
+
+main() {
```

Comment:
@kwalrath: `main()` -> `void main()` (to avoid an analyzer issue)?

Reply:
FYI, I added `void` in a recent PR, matching how we declared the `alarm()` function. If we've decided that to promote the declaration of `main()` w/o it's type, I'll revert the change.